megamek/src/megamek/common/SpecialHexDisplay.java

```diff
@@ -220,6 +267,13 @@ public Type getType() {
         return type;
     }
 
+    public Image getDefaultImage() {
+        if (type.useFolderStructure()) {
+            return type.getImage(imageSignature);
+        }
+        return type.getDefaultImage();
+    }
```

Comment:
allows to access files in folders, very handy for images that are split in multiple pieces